### PR TITLE
media-gfx/geeqie: add geeqie-9999.ebuild

### PR DESCRIPTION
**Pull request by grumpytetra**
Note: An upstream pull request has been created to rename configure.in to configure.ac

### CODE REVIEW

File: media-gfx/geeqie/geeqie-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,91 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit autotools xdg-utils git-r3
```
Comment (Zlogene) [inline media-gfx/geeqie/geeqie-9999.ebuild]: make an empty line after EAPI

File: media-gfx/geeqie/geeqie-9999.ebuild [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+DESCRIPTION="A lightweight GTK image viewer forked from GQview"
+HOMEPAGE="http://www.geeqie.org"
+EGIT_REPO_URI="https://github.com/BestImageViewer/geeqie"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64 ~ppc x86"
```
Comment (Zlogene) [inline media-gfx/geeqie/geeqie-9999.ebuild]: do not commit straight to stable

File: media-gfx/geeqie/geeqie-9999.ebuild [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+		myconf="${myconf} --enable-exiv2"
+	else
+		myconf="${myconf} --disable-exiv2"
+	fi
+
+	econf ${myconf}
+}
+
+src_install() {
+	emake DESTDIR="${D}" install
```
Comment (Zlogene) [inline media-gfx/geeqie/geeqie-9999.ebuild]: call it via `default`

File: media-gfx/geeqie/geeqie-9999.ebuild [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+	rm -f "${D}/usr/share/doc/${PF}/COPYING"
+	# Application needs access to the uncompressed file
+	docompress -x /usr/share/doc/${PF}/README.md
+}
+
+pkg_postinst() {
+	xdg_desktop_database_update
+
+	elog "Some plugins may require additional packages"
```
Comment (Zlogene) [inline media-gfx/geeqie/geeqie-9999.ebuild]: you may want using `optfeature` from `eutils` eclass

Comment (grumpytetra) [reply media-gfx/geeqie/geeqie-9999.ebuild]: Thanks. This up to code now?

File: media-gfx/geeqie/geeqie-9999.ebuild (hunk 2 above)
Comment (juippis) [reply media-gfx/geeqie/geeqie-9999.ebuild]: (Hint @grumpytetra: it's a live ebuild, they don't have keywords. That's why the CI bot is unsatisfied, too)

Comment (grumpytetra) [reply media-gfx/geeqie/geeqie-9999.ebuild]: yeah, I thought as much. I do see a fair number of other live ebuilds with keywords in the tree though.

**Comment by voyageur** [comment]
Hi @grumpytetra why did you close the PR? I do not see the live ebuild in https://packages.gentoo.org/packages/media-gfx/geeqie (sorry a bit slow in my Gentoo reviews these days)

I will probably add live ebuild based on your changes next week, if you either update the PR before that please squash in a single commit, it will be then ready for tree inclusion :)

**Comment by grumpytetra** [comment]
Hi @voyageur, no problem. I received no further word on this for a while so I thought it best not to impose.

Can't reopen since I deleted the fork. New pull request is here: #10609

**Pull request closed without merging**